src/inject/utils/dom.ts

```diff
@@ -199,12 +199,32 @@ export function removeDOMReadyListener(listener: () => void) {
     readyStateListeners.delete(listener);
 }
 
+// `interactive` can and will be fired when their are still stylesheets loading.
+// We use certain actions that can cause a forced layout change, which is bad.
+export function isCompleteDomReady() {
+    return document.readyState === 'complete';
+}
+
+export const completeDomListeners = new Set<() => void>();
```

Comment:
Should it be exported? Use `DOM` in the middle of the name and `dom` in the beginning. Also the word order seems wrong, "complete" at the beginning sounds like a verb.